speech-to-speech/SpeechToSpeech/SpeechToSpeech/AppDelegate.swift

```diff
@@ -0,0 +1,37 @@
+//
```

Comment:
The folder nesting should be simplified, got `speech-to-speech/SpeechToSpeech/SpeechToSpeech`
I believe one can be easily deleted. Be good to figure out how to name it such that they aren't quite identical.

Reply:
Removed intermediate `SpeechToSpeech` folder, Now the folder path is `speech-to-speech/SpeechToSpeech/AppDelegate.swift`.